components/brave_extension/extension/brave_extension/background/events/cosmeticFilterEvents.ts

```diff
@@ -24,18 +20,6 @@ chrome.contextMenus.create({
   parentId: 'brave',
   contexts: ['all']
 })
-chrome.contextMenus.create({
-  title: getLocale('resetSiteFilterSettings'),
-  id: 'resetSiteFilterSettings',
-  parentId: 'brave',
-  contexts: ['all']
-})
-chrome.contextMenus.create({
-  title: getLocale('resetAllFilterSettings'),
-  id: 'resetAllFilterSettings',
-  parentId: 'brave',
-  contexts: ['all']
-})
```

Comment:
I've removed the "Clear CSS rules for this site" and "Clear CSS rules for all sites" options, since it's now possible to edit these rules in more granularity from `brave://adblock`.

I'm not opposed to re-implementing the "Clear CSS rules for this site" option (we could detect any rules of the form `this.site##cssSelector` and clear them), but I think that's best addressed by a follow-up if there happens to be demand for it.